examples/Gero.purs

```diff
@@ -0,0 +1,33 @@
+-- Examples of how to use the Gero wallet interface through `Contract`
```

Comment:
I think this should referring to README instead (duplicate documentation quickly becomes inconsistent). We already have instructions there about `npm run dev`. However, they do not clarify what exactly will be built.

Could you please expand the README describing the process of building examples in more detail?

Also, README mentions Nami as the only wallet supported

> you must use Nami wallet for the time being

This would not be the case after this PR is merged.